tests/expectations/tests/dynamic_loading_with_class.rs

```diff
@@ -73,20 +73,20 @@ impl TestLib {
         P: AsRef<::std::ffi::OsStr>,
     {
         let library = ::libloading::Library::new(path)?;
-        Ok(Self::from_library(library))
+        Self::from_library(library)
     }
-    pub unsafe fn from_library<L>(library: L) -> Self
+    pub unsafe fn from_library<L>(library: L) -> Result<Self, ::libloading::Error>
```

Comment:
This is a breaking change, I guess I'm ok with this as this was just introduced though, and is more flexible.